aspnet/mvc/views/forms/sample/final/Views/Home/IndexEmpty.cshtml

```diff
@@ -0,0 +1,6 @@
+﻿@model CountryViewModelEmpty
+
+<form asp-controller="Home" asp-action="IndexEmpty" method="post">
+    <select asp-for="Country" asp-items="Model.Countries"></select> 
```

Comment:
Leave the `new SelectListItem {Value = "",   Text = "<none>" }` item out of `Model.Countries` and instead:
```
<select asp-for="Country" asp-items="@Model.Countries">
  <option value="">&lt;none&gt;</option>
</select> 
```